### ISSUE TITLE
need tests

### ISSUE THREAD

**Issue by tunnckoCore** [rkusa/koa-formidable#6]
Other than that looks good.

**Comment by rkusa** [comment]
I agree. Unfortunately, I currently don't use `koa-formidable`, whiche makes it unlikely that I add tests in the near future. However, I am fine with accepting PRs, adding collaborators or moving the repo to `koajs`.
